lighthouserc.json

```diff
@@ -0,0 +1,34 @@
+{
+  "ci": {
+    "upload": {
+      "target": "temporary-public-storage"
+    },
+    "assert": {
+      "assertions": {
+        "categories:performance": ["warn", { "minScore": 0.6 }],
```

Comment:
i notice performance fluctuates, probably due to server resources allocated on each run. are the remaining 4 categories are consistent?

Reply:
yes, i notice that too, and yes probably due to the server resources allocated, even for dev.tt.io and tt.io they are different. 
So, for the PWA, our website is not PWA so i think it is irrelevant.
The other 3 are consistent, from what i have tested the other day.

Reply:
ok sure, we can remove pwa assert or leave a comment that we not using that

Reply:
ok sure